src/command-handlers/devices.ts

```diff
@@ -0,0 +1,91 @@
+import { connectAndPrepare } from '../database';
+import { deactivateDevices, listDevices, ListDevicesOutput } from '../endpoints';
+import { reset } from '../middleware';
+import { askConfirmReset } from '../utils';
+
+type OutputDevice = ListDevicesOutput['devices'][number] & {
+    isCurrentDevice: boolean;
+};
+
+export async function listAllDevices(options: { json: boolean }) {
+    const { secrets, deviceConfiguration } = await connectAndPrepare({ autoSync: false });
+    if (!deviceConfiguration) {
+        throw new Error('Require to be connected');
+    }
+    const listDevicesResponse = await listDevices({ secrets, login: deviceConfiguration.login });
+    const result: OutputDevice[] = listDevicesResponse.devices.map(
+        (c) => <OutputDevice>{ ...c, isCurrentDevice: c.deviceId === secrets.accessKey }
```

Comment:
```suggestion
        (device) => <OutputDevice>{ ...device, isCurrentDevice: device.deviceId === secrets.accessKey }
```

use full variables names pls